Makefile

```
@@ -73,3 +81,31 @@ $(BUILTIN_ACTORS_BUNDLE): $(BUILTIN_ACTORS_CODE)
 	git pull && \
 	rustup target add wasm32-unknown-unknown && \
 	cargo run --release -- -o output/$(shell basename $@)
+
+
+# Compile the ABI artifacts and Rust bindings for the IPC Solidity actors.
+ipc-actors-abi: $(IPC_ACTORS_ABI)
+	cargo build --release -p fendermint_vm_ipc_actors
+
+# Clone the IPC Solidity actors if necessary and compile the ABI, putting down a marker at the end.
+$(IPC_ACTORS_ABI): $(IPC_ACTORS_CODE) | forge
+	if [ ! -d $(IPC_ACTORS_DIR) ]; then \
+		mkdir -p $(IPC_ACTORS_DIR) && \
+		cd $(IPC_ACTORS_DIR) && \
+		cd .. && \
+		git clone https://github.com/consensus-shipyard/ipc-solidity-actors.git; \
+	fi
+	cd $(IPC_ACTORS_DIR) && \
+	git checkout fm-156-ipc-solidity-actors && \
```

Should we point to main or not checkout at all? (I guess you are doing this for testing, right?)
```suggestion
	git checkout main&& \
```

---

Oh, you merged the fix?

---

As far as I see the 3 methods are still in `IGateway.sol`, because this hasn't been merged: https://github.com/consensus-shipyard/ipc-solidity-actors/pull/138

Until that's merged, this branch here has the 3 methods you mentioned commented out, so the size stays below 25kB and doesn't fail the entire build.